deb_files/hornet.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Hornet
+
+[Service]
+StateDirectory=hornet
+WorkingDirectory=/var/lib/hornet
+Restart=always
+Timeoutsec=Infinity
+ExecStart=/bin/bash -c "/usr/bin/hornet_download_snapshot.sh && /usr/bin/hornet -c /etc/hornet/hornet.conf"
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
I had some ideas for the service file.
Use environment file makes it easy to configure some values (no need to issue `systemctl daemon-reload` on each change, as you would to the service file)
```
[Unit]
Description=Hornet
Wants=network-online.target
After=network-online.target

[Service]
EnvironmentFile=-/etc/default/hornet
# Use a variable from the environment file, e.g:
TimeoutStopSec=${TIMEOUT_STOP_SEC}
```
Maybe instead of:
```
ExecStart=/bin/bash -c "/usr/bin/hornet_download_snapshot.sh && /usr/bin/hornet -c /etc/hornet/hornet.conf"`
```
do:
```
ExecStartPre=-/bin/bash /usr/bin/hornet_download_snapshot.sh
ExecStart=/path/to/bin/hornet $OPTIONS # options can be configured in the environment file, should the user want to add
```

Perhaps add (and pre-create the user group so not to run as root):
```
User=hornet
Group=hornet
WorkingDirectory=/path/to/hornet/workdir
```

Add security, logging, optional open files limits:
```
LimitNOFILE=4096
StandardOutput=syslog
StandardError=syslog
SyslogIdentifier=hornet

PrivateDevices=yes
PrivateTmp=yes
ProtectSystem=full
ProtectHome=yes
```